app/bundles/LeadBundle/Entity/LeadList.php

```diff
@@ -271,8 +271,8 @@ public function getFilters()
      */
     public function setIsGlobal($isGlobal)
     {
-        $this->isChanged('isGlobal', $isGlobal);
-        $this->isGlobal = $isGlobal;
+        $this->isChanged('isGlobal', (bool) $isGlobal);
```

Comment:
IsGlobal and isPreferenceCenter properties are not nullable so we can retype to bool safely here.